frontend/src/components/status-badge.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { type ReactNode } from "react";
+
+type TrackerColor = "red" | "green" | "yellow";
+
+interface StatusBadgeProps {
+  color: TrackerColor;
+  children: ReactNode;
+}
+
+export const StatusBadge: React.FC<StatusBadgeProps> = ({
```

Comment:
don't use `React.FC`, use regular functions
```suggestion
export function StatusBadge({ color, children}: StatusBadgeProps) {
```